### DISCUSSION TITLE
error in publishing npm package

### DISCUSSION THREAD

**Discussion by miminerd** [npm/feedback#255] [CLI] [answered]
Good day 👋,

I got this error when I try to publish my npm package, though I published before and never happened.

**npm ERR! 403 In most cases, you or one of your dependencies are requesting
npm ERR! 403 a package version that is forbidden by your security policy.**

this was solved by specifying 'version ' in package.json in one case or confirming the email registry in another, ...
the same error message was sent to different types of errors; if it possible can you please specify a more detailed log err message.

Thank you and have a nice day

**Comment by MylesBorins** [comment]
Hey @miminerd this appears to be a duplicate of https://github.com/npm/feedback/discussions/238#discussioncomment-407175

Did you end up opening an issue on the npm CLI repo?

**Reply by miminerd** [reply]
Hey @MylesBorins  no, I opened as improvement feature

**Reply by MylesBorins** [reply]
@miminerd thanks for clarifying. The proper place to request this is as an issue on the npm CLI repo. Often we intake requests through this discussion form and forward them to the appropriate place. In this case [the CLI issue tracker](https://github.com/npm/cli/issues) seems like the best place to request this change and get it prioritized by the team.

For larger feature requests I would normally redirect folks to the [CLI RFC process](https://github.com/npm/rfcs), but I don't think a request for better error messages requires an RFC.

Let me know if you have other questions

**Reply by MylesBorins** [reply]
I guess an additional question. What tool are you using to publish?

I just attempted to publish a package with `version` defined in the package.json and the CLI gave me this error

```
npm ERR! package.json requires a valid "version" field

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mylesborins/.npm/_logs/2021-03-04T14_44_24_915Z-debug.log
```

Could you also expand on what you mean by "confirming the email registry in another"? Are you publishing to other registries? Were you not logged in?

**Comment by MylesBorins** [comment, marked as answer]
@miminerd did you ever solve this? If you are still having issues please open a ticket with https://npmjs.com/support

**Comment by sanjarcode** [comment]
@MylesBorins see https://github.com/npm/cli/issues/4519#issuecomment-1416964139
